superset-frontend/src/views/CRUD/hooks.ts

```diff
@@ -269,7 +269,7 @@ export function useSingleViewResource<D extends object = any>(
   );
 
   const createResource = useCallback(
-    (resource: D) => {
+    (resource: D, hideToast = false) => {
```

Comment:
we need this for `updateResource` as well and make sure you are setting it the value true in `DatabaseModal/index.ts`